_posts/en/newsletters/2022-11-02-newsletter.md

```diff
@@ -0,0 +1,243 @@
+    spender could use a mechanism similar to [spontaneous
+    payments][topic spontaneous payments] where each routing node is paid
+    one sat even if the ultimate payment fails.  The spender could then
+    identify which hop the payment failed at by comparing how many
+    satoshis it sent to how many satoshis it received back.
+
+- **Anchor outputs workaround:** Bastian Teinturier [posted][teinturier
+  fees] to the Lightning-Dev a [proposal][bolts #1036] for replacing
+  [anchor outputs][topic anchor outputs] with multiple presigned
+  versions of each [HTLC][topic htlc] at different feerates.  Anchor
+  outputs were introduced with the development of the [CPFP
+  carve-out][topic cpfp carve out] rule for allowing fees to be added to
+  a transaction via the [CPFP][topic cpfp] mechanism in a way that
+  wouldn't be [pinnable][topic transaction pinning] for LN's two-party
+  contract protocol.  However, Teinturier [notes][bolts #845] that using
+  CPFP requires every LN node keep a pool of non-LN UTXOs ready to spend
+  at any moment.  By comparison, presigning multiple versions of HTLCs
+  each with different fees allows those fees to be paid directly from
+  the HTLC's value---no additional UTXO management is required.
+
+    He's seeking support from other LN developers for the idea of moving
+    to multiple feerate HTLCs.  All discussion as of this writing has
+    occurred on Teinturier's [PR][bolts #1036].
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 24.0 RC2][] is a release candidate for the
+  next version of the network's most widely used full node
+  implementation.  A [guide to testing][bcc testing] is available.
+
+  **Warning:** this release candidate includes the `mempoolfullrbf`
+  configuration option which several protocol and application developers
+  believe could lead to problems for merchant services as described
+  in newsletters FIXME and FIXME.  Optech encourages any services that
+  might be affected to evaluate the RC and participate in the public
+  discussion.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #23927][] restricts `getblockfrompeer` on pruned nodes
+  to heights below the node's current synchronization progress. This
+  prevents a footgun arising from retrieving future blocks making the
+  node's block-files ineligible for pruning.</br>
```

Comment:
Not sure if the <br /> (slash after) is intended or if two newlines were intended here.